packages/compass-sidebar/src/components/connections-filter-popover.tsx

```diff
@@ -102,7 +102,7 @@ export default function ConnectionsFilterPopover({
               onMouseLeave={handleButtonMouseLeave}
               active={open}
               aria-label="Filter connections"
-              ref={ref as React.Ref<unknown>}
+              ref={ref}
```

Comment:
I believe it's possible to pass the `ref` here without having to provide an explicit type parameter to the `InteractivePopover` above, is possible because the `IconButton` use the `forwardRef` utility internally.